meson.build

```diff
@@ -116,6 +120,10 @@ xkbcommon_dep = dependency('xkbcommon')
 # Endless-specific: parental controls
 eos_parental_controls_dep = dependency('eos-parental-controls-0', required: true)
 
+# Endless-specific: Custom animations
+libanimation_glib_dep = dependency('libanimation-glib-0')
+libeos_shell_fx_deps = [mutter_dep, gdk_pixbuf_dep, libanimation_glib_dep, m_dep]
```

Comment:
Not sure these dependencies are correct... GdkPixbuf does not seem to be used by the effect, and there is no use of the mutter API, perhaps we can replace it with clutter?